.github/workflows/format.yml

```diff
@@ -0,0 +1,21 @@
+name: 💅 Format and Linter checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
+jobs:
+  build:
+    name: 💅 Format and Linter checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: 🤹‍♀️ Install modules
+        run: yarn
+      - name: 💅 Run format check
+        run: yarn format
+      - name: 💅 Run lint check
+        run: yarn lint
```

Comment:
Can `yarn format (prettier)` and `yarn lint (eslint)` co-exist? i.e. can we do `yarn format:fix` and `yarn lint:fix` and satisfy both? I think we don't have to, rather we can remove `yarn lint`.
Also, in order to make this workflow "required" for PR check, we need higher permission which I don't have... the easier alternative is to add `yarn format` into existing required workflow, such as unit-tests.yaml instead.

Reply:
Yup, there might be rules that could collide, but it seems that we don't have any at the moment.

I also added the workflow as required now.